src/core/crypto/multiEd25519.ts

```diff
@@ -146,26 +192,33 @@ export class MultiEd25519Signature extends Signature {
    *
    * @param args.signatures A list of signatures
    * @param args.bitmap 4 bytes, at most 32 signatures are supported. If Nth bit value is `1`, the Nth
-   * signature should be provided in `signatures`. Bits are read from left to right
+   * signature should be provided in `signatures`. Bits are read from left to right.
+   * Alternatively, you can specify an array of bitmap positions.
+   * Valid position should range between 0 and 31.
```

Comment:
lets add this check in the constructor?

Reply:
The constructor calls `createBitmap` which already performs the check